include/sentry.h

```diff
@@ -1405,6 +1405,10 @@ SENTRY_API sentry_uuid_t sentry_capture_event(sentry_value_t event);
 
 /**
  * Allows capturing independently created minidumps.
+ * 
+ * You don't need this function if you're relying on Sentry to create the minidump.
+ * This is useful when you have a minidump captured through a different mechanism,
+ * and you want Sentry to ingest it.
```

Comment:
This is great, but I would add it as a `Note: ` paragraph at the end of the inline comment. We do this with most meta descriptions in the header.